### ISSUE TITLE
Can't run the code

### ISSUE THREAD

**Issue by voileee** [Sinwang404/DeepDDs#4]
Hello author, I'm very inspired by your work, but I can't run your code, the following error is reported:    
assert (len(xd) == len(xt) and len(xt) == len(y)), "The three lists must be the same length!"
TypeError: object of type 'NoneType' has no len()

**Comment by Sinwang404** [comment]
Hello, thank you for your interesting
I advise you to double-check the data， follow the README that we gave

**Comment by HelloWorldLTY** [comment]
Nope, even if we used the example datasets, we still received this bug.

The solution is to replace the line 96 of trianing_GCN.py code as:

```
datafile = 'new_labels_0_10_leave'
```

I believe this should be an error caused by typo, so I recommended the authors to correct it.

**Comment by whiteroad01** [comment]
> Nope, even if we used the example datasets, we still received this bug.
> 
> The solution is to replace the line 96 of trianing_GCN.py code as:
> 
> ```
> datafile = 'new_labels_0_10_leave'
> ```
> 
> I believe this should be an error caused by typo, so I recommended the authors to correct it.

In fact, labels from 'new_labels_0_10_leave' are all 1, which is weird. Therefore, replacing 'datafile' of creat_data_DC.py is a better choice.
